include/deal.II/integrators/divergence.h

```diff
@@ -451,82 +424,28 @@ namespace LocalIntegrators
     }
 
     /**
-     * The weak form of the grad-div operator penalizing volume changes
-     * @f[
-     *  \int_Z \nabla\!\cdot\!u \nabla\!\cdot\!v \,dx
-     * @f]
-     *
-     * @author Guido Kanschat
-     * @date 2011
+     * @deprecated Use LocalIntegrators::GradDiv::cell_matrix() instead.
      */
     template <int dim>
     void grad_div_matrix (
       FullMatrix<double> &M,
       const FEValuesBase<dim> &fe,
-      double factor = 1.)
+      double factor = 1.) DEAL_II_DEPRECATED
```

Comment:
while there, could make the last argument `const`. Or not, since it's not important. Your choice.